src/view/seed.js

```diff
@@ -38,10 +38,10 @@ const copyStyles = StyleSheet.create({
   },
   title: {
     textAlign: 'center',
-    marginBottom: 15,
   },
   copyTxt: {
     textAlign: 'center',
+    marginTop: 15,
```

Comment:
This was just something I saw in the SeedView. We should always add `topMargin` to the following element. I think I added this the wrong way and you did it right here.

Reply:
ah yeah good to be consistent about that